### PR TITLE
update conclusion

### PR DESCRIPTION
**Pull request by dmurvihill**

### CODE REVIEW

File: doc/src/conclusion.ltx [hunk 1]
```diff
@@ -7,26 +7,24 @@ add so many features which we felt might be valuable. There is much work that
...
-Vector-Distance approaches (like \textit{Smith-Waterman}) while being
-substantially faster (for details, see the description of MOSS in
+\textit{Checksims} is the ability to use fingerprinting algorithms with a
+persistent database of student submissions; such an improvement would allow
+fast checking not only within a single assignment (as \textit{Checksims} does
+today), but also against every assignment previously run through the detector
+--- detecting students who, for example, used another student's assignment
+from a previous year. Furthermore, advanced fingerprinting algorithms offer
+most of the accuracy of Vector Distance approaches (like
+\textit{Smith-Waterman}) while being substantially faster for details, see
```
Comment (mheon) [inline doc/src/conclusion.ltx]: Missing a paren before for details

File: doc/src/conclusion.ltx [hunk 2]
```diff
@@ -42,21 +40,18 @@ cases of similarity substantially.
...
 Usability improvements for \textit{Checksims} could also come from tools to
 make it easier to apply to assignments. Professor Lauer has suggested
-integration with online homework submission systems (such as \textit{Turnin})
-which would run a \textit{Checksims} scan automatically after submissions for
-an assignment are closed. Another option would be a GUI wrapper for
-\textit{Checksims} to automate common tasks (for example, placing student
-submissions into separate folders, or decompressing submissions given as .zip
-or .tar files).
+integration with online homework submission systems such as the most popular
+in-house system used at WPI, which could run a \textit{Checksims} scan
```
Comment (mheon) [inline doc/src/conclusion.ltx]: Are we not saying what that system (Turnin) is? If we're mentioning it, it feels inappropriate if we DON'T namedrop it.

Comment (dmurvihill) [inline doc/src/conclusion.ltx]: Lauer said if we namedrop it, we have to cite it. And there's not really anything to cite. (why do we have to cite it? who knows...)

Comment (mheon) [inline doc/src/conclusion.ltx]: Maybe say less about it then? Don't even mention that it's popular or in-house, just it's the one used at WPI?

**Comment by dmurvihill** [comment]
Okay, took care of your comments.

File: doc/src/conclusion.ltx (hunk 2 above)
Comment (dmurvihill) [inline doc/src/conclusion.ltx]: New commit. This works for me if it works for you.

**Pull request merged**